pkg/services/accesscontrol/models.go

```diff
@@ -396,7 +396,6 @@ const (
 
 	// Settings scope
 	ScopeSettingsAll  = "settings:*"
-	ScopeSettingsAuth = "settings:auth:*"
```

Comment:
Thanks for pointing that out! I added `ScopeSettingsAuth` accidentally to the `fixed:authentication.config:writer` role in https://github.com/grafana/grafana/commit/5285e9503be5702680acb2b52a6bda0632f4603d#diff-16771e4ac815676323055b0b70f4d742119be71d2ffed19b3de7a8059a6f1752 and now I'm reverting that + removing the constant because it's not used anywhere in the code.